dependencies/linux/install-dependencies-focal

```diff
@@ -73,8 +73,8 @@ sudo apt-get -y install \
 # Java 8 (not in official repo for focal)
 sudo add-apt-repository -y ppa:openjdk-r/ppa
 sudo apt-get update
-sudo apt-get -y install openjdk-8-jdk
-sudo update-alternatives --set java /usr/lib/jvm/java-8-openjdk-amd64/jre/bin/java
+sudo apt-get -y install openjdk-11-jdk
```

Comment:
Does not seem related to the intent -- was this included intentionally?

Reply:
Yes, I had to change this to get the ant build working. Would you prefer I exclude it?